templates/queue.html

```diff
@@ -328,5 +329,30 @@ <h1>
         modalMessage.innerHTML = message;
         modal.style.display = "block";
     });
+
+    rollupContinueButton.addEventListener("click", () => {
+        const scopes = ["public_repo", "workflow"];
+
+        // Gather PR numbers
+        let selectedRows = table.rows({ selected: true }).nodes().toArray();
+        let nums = selectedRows
+            .map(row => {
+                let numberCell = row.cells[1];
```

Comment:
Can we again encode the pull request number with a `data-` attribute on the given cell? To avoid parsing it needlessly from a link.